bpf/bpf_lxc.c

```diff
@@ -1579,12 +1579,8 @@ ipv6_policy(struct __ctx_buff *ctx, struct ipv6hdr *ip6, __u32 src_label,
 			/* This is a reply, the proxy port does not need to be embedded
 			 * into ctx->mark and *proxy_port can be left unset.
```

Comment:
the comment about `*proxy_port` becomes outdated by this change

Reply:
== why is this needed now?
```
			*proxy_port = 0;
```

Reply:
The trace notification uses the `proxy_port`, it felt robuster to demonstrate what value it will take in this path.